docs(router): Clarify how to clear secondary routes (#40272)
angular/angular: aio/content/guide/router-tutorial-toh.md

That's not a route, but it is a legitimate value.
Setting the popup `RouterOutlet` to `null` clears the outlet and removes the secondary popup route from the current URL.

<div class="alert is-critical">

**Note:** All commands in the array passed to `Router.navigate()` target a _specific segment_ in the `UrlTree`. 
We specify the parent of the `ActivatedRoute` as the `relativeTo` option because we want to remove `'popup'` from the segment which holds its reference.
It's important to always be aware of which segments the commands will be applied to.

</div>

<div class="alert is-helpful">

> When `relativeTo` is not provided to the `Router.navigate()` method, the commands are processed starting at the root.

Reviewer: ```suggestion
When you do not provide `relativeTo` to the `Router.navigate()` method, Angular processes the commands starting at the root.
```

I'm guessing at the subjects here ("you" and "Angular"), but basically you just want to avoid the passive voice and provide the subject for each clause.

> We could omit the `relativeTo` option in this particular example because the `'popup'` outlet appears at the root level of the configuration.

Reviewer: ```suggestion
You could omit the `relativeTo` option in this particular example because the `'popup'` outlet appears at the root level of the configuration.
```

  
</div>

<div class="alert is-helpful">

> If you want to close an outlet which appears at any segment depth, you could accomplish
> this by creating a `UrlTree` from the current URL, recursively clearing segment `children` matching the outlet name, and finally 
> calling `Router.navigateByUrl()` with the `root` segment of the current `UrlTree`.

Reviewer: ```suggestion
To close an outlet which appears at any segment depth, create a `UrlTree` from the current URL, recursively clear segment `children` matching the outlet name, and call `Router.navigateByUrl()` with the `root` segment of the current `UrlTree`.
```
